manager/src/test/java/io/gingersnapproject/mysql/MySQLResources.java

```diff
@@ -30,10 +30,10 @@ public Map<String, String> start() {
       db.start();
 
       Map<String, String> properties = new HashMap<>( Map.of(
-            "quarkus.datasource.db-kind", "MYSQL",
-            "quarkus.datasource.username", db.getUsername(),
-            "quarkus.datasource.password", db.getPassword(),
-            "quarkus.datasource.reactive.url", String.format("mysql://%s:%d/debezium", db.getHost(), db.getMappedPort(MySQLContainer.MYSQL_PORT)),
+            "quarkus.datasource.mysql.db-kind", "MYSQL",
```

Comment:
Could we make the `db-kind` implicit or is it needed at build time?

Reply:
Technically this line is not needed as it is a build time only property, so it is already implicitly defined. I will remove the line.